korangar/src/loaders/animation/mod.rs

```diff
@@ -297,9 +319,8 @@ impl AnimationLoader {
 /// inside the frame bounding rectangle with vertices [-1, 0], [-1, 2], [1, 0],
 /// [1, 2].
 fn convert_coordinates(coordinates: Vector2<i32>, size: Vector2<i32>) -> Vector2<f32> {
-    const EPSILON: f32 = 0.0001;
-    let x = (coordinates.x as f32 / size.x as f32 - 0.5) * 2.0 + EPSILON;
-    let y = 2.0 - (coordinates.y as f32 / size.y as f32) * 2.0 + EPSILON;
+    let x = (coordinates.x as f32 / size.x as f32 - 0.5) * 2.0;

```

Comment:
The old and new code don't properly guard against division by zero (if either dimension of "size" is 0).

Reply:
Panic when the size is 0, the construction doesn't allow, but for debugging purposes would be great!